2-mandatory/2-financial-times.js

```diff
@@ -14,25 +21,51 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+   const separateTitleArray = [];
+    for (const articleTitle of allArticleTitles) {
+       allArticleTitles.toString().split(" ");
+        separateTitleArray.push(articleTitle)
+    }
+        let titleWithFewestWords = separateTitleArray.reduce((fewestWords, currentWords) => {
```

Comment:
This part of the code looks very good - but there's one small problem.
The question is asking you for the title with the fewest words - but here it looks like you're comparing the number of characters: `currentWords.length < fewestWords.length`
Can you think about how you might fix this?